Dockerfile

```diff
@@ -57,6 +54,7 @@ RUN apt-get update && \
         mysql-client \
         poppler-utils \
         sudo \
+        vim \
```

Comment:
I think none should be installed by default.  Either people can edit files in their host OS and see them changed in the running container thanks to volume mounting, or they can do something like `docker exec -i -t --user root myapp_web_1 /bin/bash` and, when inside, then `apt-get install vim` as usual.